content/6_Advanced/String_Search.mdx

```diff
@@ -99,7 +99,24 @@ vector<int> pi(const string &s) {
 ```
 
 </CPPSection>
+<PySection>
 
+```py
+
+def pi(s: str) -> List[int]:
+	n: int = len(s)
+	pi_s: List[int] = [0] * n
+	j: int = 0
```

Comment:
uh no need for type annotations for these variables

Reply:
ok, I'll remove them